schema/vrs.json

```diff
@@ -588,17 +588,17 @@
             "comparator"
          ]
       },
-      "Range": {
+      "IntegerRange": {
          "description": "A bounded, inclusive range of integers.",
```

Comment:
Are we adding any checks/constraints to assure that min is less than max? Also, we may want to state that if min=max that it will be considered equivalent to SimpleInteger.

Reply:
JSON Schema does not support relative value constraints. But I agree we should add this constraint to our text definition & documentation.